src/app/public/modules/list-view-checklist/list-view-checklist.module.ts

```diff
@@ -34,8 +30,7 @@ import {
   imports: [
     CommonModule,
     FormsModule,
-    SkyCheckboxModule,
-    SkyListViewChecklistResourcesModule
```

Comment:
Removed the resources module because it wasn't being used, and is reducing code coverage.